test/mock_server/server.js

```diff
@@ -75,11 +78,45 @@ app.use('/v2', v2Router);
 app.use('/external', external);
 
 // Attaching V2 routers
+v2Router.use('/acts/redirect-actor-id', async (req, res) => {
+    res.json({ data: { name: 'redirect-actor-name', id: 'redirect-run-id' } });
+});
 v2Router.use('/acts', actorRouter);
 v2Router.use('/actor-builds', buildRouter);
+v2Router.use('/actor-runs/redirect-run-id/log', async (req, res) => {
+    // Asynchronously write each chunk to the response stream
+    for (const chunk of MOCKED_ACTOR_LOGS) {
+        res.write(chunk);
+        res.flush(); // Flush the buffer and send the chunk immediately
+        // Wait for a short period to simulate work being done on the server
+        await new Promise((resolve) => {
+            setTimeout(resolve, 10);
+        });
+    }
+
+    // End the response stream once all chunks have been sent
+    res.end();
+});
+v2Router.use('/actor-runs/redirect-run-id', async (req, res) => {
+    res.json({ data: { id: 'redirect-run-id', actId: 'redirect-actor-id', status: 'SUCCEEDED' } });
+});
 v2Router.use('/actor-runs', runRouter);
 v2Router.use('/actor-tasks', taskRouter);
 v2Router.use('/users', userRouter);
+v2Router.use('/logs/redirect-log-id', async (req, res) => {
```

Comment:
You can store the handler arrow function as a variable and reuse it in both `/actor-runs/redirect-run-id` and `/logs/redirect-log-id`. 

Alternatively, some `express` versions accept string arrays and / or regexps for paths, but some of those might be deprecated.